Tutorials/Making_Maps_with_Cartopy.ipynb

```diff
@@ -18,21 +18,14 @@
    "cell_type": "code",
```

Comment:
<p>I would add here that it is probably better in Antarctica (or we recommend) to plot with the model mask which is different to cartopy's land (as explained in the plot model mask section). I.e., make it more clear that we side with model mask vs cartopy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Sure. Why is it preferred in Antarctica though, I don't really know deep inside.

Reply:
Because the model mask is very different from the cartopy land mask near Antarctica, due to the differences where the ice shelves are.

Reply:
thanks!

Reply:
<p>I added it bit further down.</p>